docs/console/tutorials/configure-marketplace-components/api-portal.mdx

```diff
@@ -57,10 +57,9 @@ This application include the following components:
 - you can choose to create a new _API Gateway_ service or choose an already installed service. 
 
 and the following endpoints:
-- _/documentations/api-portal_ to the [_API Portal_](/runtime_suite/api-portal/10_overview.md) service 
-- _/documentations/api-portal/api_ to the [_Swagger Aggregator_](/runtime_suite/swagger-aggregator/10_overview.md) service
-- _/documentations/openapi_ to the [_Swagger Aggregator_](/runtime_suite/swagger-aggregator/10_overview.md) service
-- _/documentations/swagger_ to the [_Swagger Aggregator_](/runtime_suite/swagger-aggregator/10_overview.md) service
+- _/documentations/api-portal_ to the [_API Portal_](/runtime_suite/api-portal/10_overview.md) service
+- _/api/openapi_ to the [_Swagger Aggregator_](/runtime_suite/swagger-aggregator/10_overview.md) service
```

Comment:
is there any path rewrite to be set or is `/` enough? Should it be specified?

Reply:
I added a few info about the endpoint that has a rewrite but I don't think it's necessary to add more details about the others because in this tutorial, the user doesn't have to create the endpoints on its own